.github/workflows/check-pypi.yml

```diff
@@ -0,0 +1,28 @@
+
+name: Check if required secrets are set to publish to Pypi
+
+on: push
+
+jobs:
+  checksecret:
+    name: check if PYPI_TOKEN and TESTPYPI_TOKEN are set in github secrets
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check PYPI_TOKEN
+        env:
+          PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
+        run: |
+          if ${{ env.PYPI_TOKEN == '' }} ; then
+            echo "PYPI_TOKEN secret is not set"
+            exit 1
+          fi
+      - name: Check TESTPYPI_TOKEN
+        env:
+          TESTPYPI_TOKEN: ${{ secrets.TESTPYPI_TOKEN }}
```

Comment:
Nit: TEST_PYPI_TOKEN is better than TESTPYPI token.

Reply:
@chayim I recommend keeping TESTPYPI given we use that naming on the other projects

Reply:
Fair. This is a matter of personal preferences - not a religious war here ;)